code/core/src/basiselements/RemovableElement.java

```diff
@@ -0,0 +1,16 @@
+package basiselements;
+
+/**
+ * An object that is not removable as default.
+ *
+ * <p>Must be implemented for all objects that should be controlled by the <code>EntityController
+ * </code>.
+ */
+public interface RemovableElement {
```

Comment:
Wir sollten uns schon etwas an die Java Naming-Conventions halten. 

```suggestion
public interface Removeable{
```

Reply:
es wird abstrakte Klasse